source/mir/ndslice/topology.d

```diff
@@ -3067,3 +3069,411 @@ unittest
     assert(lowerData == [0, 3, 4, 6, 7, 8]);
     assert(upperData == [0, 1, 2, 4, 5, 8]);
 }
+
+// Transposed adjusted to ignore dim=0 and include universal
+private template adjTransposed(size_t dim = 0)
+{
+    import mir.ndslice : Slice, SliceKind;
+	
+    auto adjTransposed(SliceKind kind, size_t[] packs, Iterator)(Slice!(kind, packs, Iterator) slice)
+    {
+        static assert (packs != [1], "1-dimensional slice not accepted");
```

Comment:
@9il I had only included that initially because transposed does not work properly on vectors, i.e.
```
auto slice = iota(3);
auto x = slice.transposed!1; //does not work
```
but with the adjustment for packs, then one can only pass dim=0 for a 1-dimensional vector and there is no error.